commit ed10150657b1f3ae4f061c751daf365ed16ca8d5
Author: Trent Mick <trentm@gmail.com>
Date:   2019-06-10T12:06:22-07:00 (4 months ago)
    
    TOOLS-604 want tooling to check copyrights (fix bash usage to not exit on first err)
    Reviewed by: Dave Eddy <dave.eddy@joyent.com>
    Approved by: Dave Eddy <dave.eddy@joyent.com>

diff --git a/tools/check-copyright b/tools/check-copyright
index 4d38fef..b260b93 100755
--- a/tools/check-copyright
+++ b/tools/check-copyright
@@ -90,7 +90,7 @@ function check_file {
     # Wrong copyright year?
     if [[ "$hit" != *$year* ]]; then
         echo "error: $file: copyright year not updated to $year: '$hit'" >&2
-        (( numErrors++ ))
+        numErrors=$(( numErrors + 1 ))
         return
     fi
 
@@ -100,11 +100,11 @@ function check_file {
         if [[ -n "$optWarnOnlyOnFormat" ]]; then
             echo "warning: $file: copyright not in RFD 164 form, '$hit'" \
                 "does not match /$blessed/" >&2
-            (( numWarnings++ ))
+            numWarnings=$(( numWarnings + 1 ))
         else
             echo "error: $file: copyright not in RFD 164 form, '$hit'" \
                 "does not match /$blessed/" >&2
-            (( numErrors++ ))
+            numErrors=$(( numErrors + 1 ))
         fi
         return
     fi
